### DISCUSSION TITLE
Traduction

### DISCUSSION THREAD

**Discussion by Azga** [cbartel/nw-company-tool#46] [General]
Hello,

What can I do if I want to make a trad of everything in another language ?

Thanks in advance.

**Comment by cbartel** [comment]
Hey Azga,
sadly I have stopped working on this for a while, as I'm not playing New World any more ..

There are some things to do for a translation.

I have prepared the services but did not finish the work.
You can find it here
https://github.com/cbartel/nw-company-tool/tree/main/webapp/src/assets/i18n

an example of how these are used can be found here
https://github.com/cbartel/nw-company-tool/blob/648e4a5f313bb2e62218c437f6d65d14bb30a6a2/webapp/src/app/pages/my-character/components/character-attributes/character-attributes.component.html#L2

The used translation is set in the translation module, the default is set to en, which maps to the file en.json in the i18n directory (see link above)
https://github.com/cbartel/nw-company-tool/blob/648e4a5f313bb2e62218c437f6d65d14bb30a6a2/webapp/src/app/app.module.ts#L60

if you were to make a translation, you would need to check all .html files for the content you want to translate, replace the english text with a placeholder followed by the translate pipe 
`{{'TRANSLATEKEYHERE' | translate}` and define all the keys in your file in the i18n directory.


Hope this helps :)

**Reply by Azga** [reply]
Thanks I will checking this.
